app/controllers/application_controller.rb

```diff
@@ -9,5 +9,10 @@ class ApplicationController < ActionController::Base
 
  def configure_permitted_parameters
    devise_parameter_sanitizer.for(:sign_up) << [:username, :crew_tag]
+   devise_parameter_sanitizer.for(:account_update) << [
+    :username,
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.